src/ensembl/jest.config.js

```diff
@@ -16,9 +16,7 @@ module.exports = {
   },
   preset: 'ts-jest',
```

Comment:
I am still a bit confused though. If this PR still uses ts-jest, then what was the point in updating babel.conf? If, however, we switch to babel for tests, then shouldn't we also make changes here?

Reply:
Yes, we should remove `ts-jest`.